packages/code-du-travail-modeles/src/__test__/common/travailleur-handicape/preavis-retraite/depart-mise.spec.ts

```diff
@@ -11,7 +11,7 @@ describe("Travailleur handicapé - Depart et mise à la retraite", () => {
     ${6}      | ${2}
     ${24}     | ${3}
   `(
-    "En ayant $seniority mois d'ancienneté, son préavis devrait être $expectedNotice mois",
+    "Mise à la retraite: en ayant $seniority mois d'ancienneté, son préavis devrait être $expectedNotice mois",
```

Comment:
J'aurais plus mis dans le sens de la phrase: 
```suggestion
    "En ayant $seniority mois d'ancienneté, son préavis de mise à la retraitre devrait être $expectedNotice mois",
```